videosink/format.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package videosink
+
+import "fmt"
+
+type ImageFormat int
+
+const (
+	PNG ImageFormat = iota
+	JPEG
+
+	// DefaultFormat is the format used when not set explicitly in options or
+	// as a URL parameter.
+	DefaultFormat = PNG
+)
+
+func (f ImageFormat) String() string {
```

Comment:
I think I'd prefer to use stringer; golang.org/x/tools/cmd/stringer

Reply:
I gave it a spin, only to realize that the `mimeType()` and `ImageFormatFromString` functions also need to be maintained when changing formats. As such I think stringer would add plenty of unnecessary complexity.